cluster-autoscaler/main.go

```diff
@@ -211,6 +211,7 @@ var (
 	maxScaleDownParallelismFlag        = flag.Int("max-scale-down-parallelism", 10, "Maximum number of nodes (both empty and needing drain) that can be deleted in parallel.")
 	maxDrainParallelismFlag            = flag.Int("max-drain-parallelism", 1, "Maximum number of nodes needing drain, that can be drained and deleted in parallel.")
 	gceExpanderEphemeralStorageSupport = flag.Bool("gce-expander-ephemeral-storage-support", false, "Whether scale-up takes ephemeral storage resources into account for GCE cloud provider")
+	gceReservationsEnabled             = flag.Bool("gce-reservations-enabled", false, "Include the GCE Reservations when making the scale-up decisions.")
```

Comment:
I this this is a vendor specific behavior, should we put this here, or move to `cloudprovider/gce` ?